src/app/components/shared/delete-modal/delete-modal.component.ts

```diff
@@ -0,0 +1,49 @@
+import { Component, Input } from "@angular/core";
+import { PageComponent } from "@helpers/page/pageComponent";
+import { withUnsubscribe } from "@helpers/unsubscribe/unsubscribe";
+import { ModalComponent } from "@menu/widget.component";
+import { SharedActivatedRouteService } from "@services/shared-activated-route/shared-activated-route.service";
+
+@Component({
+  selector: "baw-delete-modal",
+  template: `
+    <div class="modal-header">
+      <h4 class="modal-title">Delete Item</h4>
+      <button
+        type="button"
+        class="btn-close"
+        aria-label="Close"
+        (click)="dismiss()"
+      ></button>
+    </div>
+
+    <div class="modal-body">
+        <span id="subTitle">
+          <p>Are you certain you wish to delete this item?</p>
+        </span>
+    </div>
+
+    <div class="modal-footer">
+      <a class="btn btn-outline-primary" (click)="dismiss()">Cancel</a>
+      <a class="btn btn-danger text-white" (click)="deleteModel()">Delete</a>
+    </div>
+  `,
+})
+export class DeleteModalComponent extends withUnsubscribe() implements ModalComponent {
+  public constructor(
+    private sharedRoute: SharedActivatedRouteService,
```

Comment:
hmmmm code smell

Reply:
The `sharedRoute` is no longer needed and I have therefore been able to remove the constructor